### ISSUE TITLE
Installation on Ubuntu

### ISSUE THREAD

**Issue by ampaschoal** [leoliuf/MRiLab#15]
Hi,

I am trying to install MRiLab on Ubuntu. I could to compile all itens but not the ISMRMRD. In fact, I could to compile it, but when I do the cmake ../ command in <path>/MRiLab/Lib/src/build  the following error occur.

Then I noted that the file ismrmrd.hxx does not exist. It wasn't generated.

Can anybody imagine what I am doing wrong?

ISMRMRD and Boost Found, MRiLab to Gadgetron interface will be compiled
CMake Error: The following variables are used in this project, but they are set to NOTFOUND.
Please set them or make sure they are set and tested correctly in the CMake files:
ISMRMRD_XSD_INCLUDE_DIR
   used as include directory in directory /home/apaschoal/Documents/MRiLab/Lib/src/interface/DoMatToHDF5
   used as include directory in directory /home/apaschoal/Documents/MRiLab/Lib/src/interface/DoMatToHDF5
   used as include directory in directory /home/apaschoal/Documents/MRiLab/Lib/src/interface/DoMatToHDF5
   used as include directory in directory /home/apaschoal/Documents/MRiLab/Lib/src/interface/DoMatToHDF5
   used as include directory in directory /home/apaschoal/Documents/MRiLab/Lib/src/interface/DoMatToHDF5
   used as include directory in directory /home/apaschoal/Documents/MRiLab/Lib/src/interface/DoMatToHDF5
   used as include directory in directory /home/apaschoal/Documents/MRiLab/Lib/src/interface/DoMatToHDF5

Thank you,

André.

**Comment by leoliuf** [comment]
Please check if you can find ismrmrd.hxx in the ISMRMRD install path
Please see FindIsmrmrd.cmake
```
#   ISMRMRD_XSD_INCLUDE_DIR  - folder containing ismrmrd.hxx (autogenerated from xsd schema)
FIND_PATH( ISMRMRD_XSD_INCLUDE_DIR ismrmrd.hxx
HINTS $ENV{ISMRMRD_HOME} PATHS /usr/local /usr PATH_SUFFIXES schema ismrmrd ismrmrd/schema)
```

**Comment by ampaschoal** [comment]
Thank you for your attention.

That is my problem. I can't find ismrmrd.hxx.

I already set the correct paths on FindIsmrmrd.cmake.

**Comment by ampaschoal** [comment]
I am using xsd version 4.0.0.7.

Sould I try older versions?

**Comment by leoliuf** [comment]
ISMRMRD is evolving. This current MRiLab version is tested using ISMRMRD 0.5.0, so it should work with XSD version supported by ISMRMRD 0.5.0.

**Comment by ampaschoal** [comment]
I'll try using ismrmrd 0.5.0 on monday in my PC in the university, but it might work now. I noted that in cmake folder of ismrmrd 0.5.0 there is the file FindXSD.cmake and findxerces. Both of this files doesn't exist in ismrmrd 1.1 version. 

I'll let you know if it work, but now I thing that will run! Thank you!

**Comment by ampaschoal** [comment]
The previous error was solved using ISMRMRD 0.5.0

However now I have others errors. Could you please tell me the version of cuda vtk and gadgetron that MRiLab is tested?

**Comment by leoliuf** [comment]
The information about the version of cuda, vtk and gadgetron that MRiLab is tested can be found in the 1.3 section (Table 1.1) of the user guide.
